pallets/xcm-helper/src/lib.rs

```diff
@@ -345,7 +429,11 @@ pub mod pallet {
 			Ok(().into())
 		}
 
-		///Set Thea Key
+		/// Initializes Thea Key.
+		///
+		/// # Parameters
+		///
+		/// * `new_thea_key`: New Key which will replace existing Key.
```

Comment:
real argument name is `thea_key` and not `new_thea_key`. Please update.

Reply:
done